docs/UserGuide.md

```diff
@@ -97,39 +97,35 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a patient’s details : `edit`
 
-Edits an existing person in the address book.
+Edits an existing patient in the patient list.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the person at the specified INDEX. The index refers to the index number shown in the displayed person list. * The index must be a positive integer 1, 2, 3, …​
```

Comment:
Minor formatting issues
Use code syntax for 'INDEX'
```
* Edits the person at the specified `INDEX`.
```
Move these to separate bullet points
```
* The index refers to the index number shown in the displayed person list.
* The index **must be a positive integer** 1, 2, 3, …​
```